### ISSUE TITLE
minor bugs magic wand

### ISSUE THREAD

**Issue by TRuikes** [tensorflow/tflite-micro-arduino-examples#165]
Hi! I've run the tutorial for the magic wand, and got it working with a few edits.

1) In the tutorial it says the labels for training data (made with the webpage) can be anything like `0` or `Z`, however the notebook will only work with intergers as labels. The quickest workaround without redoing the gestures, is to rename the at the start of the notebook:

```
import glob
import json

label_map = dict()

strokes = []
counter = 0
for filename in glob.glob("magic_wand_digit_data/*.json"):
  if '3_gestures' not in filename:
    continue

  print(filename)
  with open(filename, "r") as file:
    file_contents = file.read()
  file_data = json.loads(file_contents)
  for stroke in file_data["strokes"]:
    if stroke['label'] == 'k':
      stroke['label'] = 'l'

    if stroke['label'] not in label_map.keys():
      label_map[stroke['label']] = f'{counter}'
      counter += 1

    stroke['label'] = label_map[stroke['label']]
    stroke["filename"] = filename
    strokes.append(stroke)
```
Perhaps remove the suggestion to use lettres as labels from the tutorial.

2) the keyword `num_classes` (in `model = make_model(input_shape=(IMAGE_WIDTH, IMAGE_HEIGHT, 3), num_classes=3)
`) should but updated to match the number of classes in the input data. Perhaps a hint herefore in the notebook or tutorial is nice.

3) For small training data (i.e. 10-15 images for 1 class, as suggested), the random sampling can leavout a class when generating the trianing dataset. I had 3 classes, but it would draw example images from only 2 for the `/train` folder.  The result is that `model.fit` doesnt work. Excluding the validation step from `model.fit` allows the code to pass:
`model.fit(
    train_ds, epochs=epochs, callbacks=callbacks, )# validation_data=validation_ds,
)`

**Comment by ddavis-2015** [comment]
Thank you for your interest in TFLM, your feedback is greatly appreciated.

Notes:
splitting of the data sets for training should transition from numpy APIs to tensorflow dataset APIs
